### PR TITLE
Add documentation for serverFailoverEnable

### PR DESCRIPTION
**Pull request by bbilbrey-aveva**

### CODE REVIEW

File: content/configuration/data-source.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ The following parameters are available for configuring an OPC UA data source:
...
 | **dataCollectionMode** | Optional | `string` | Specifies the data collection mode the adapter is in. The following data collection modes are available:<br> **HistoryOnly** <sup>2</sup> - The adapter component does not get started and history recovery on-demand is enabled. For more information, see [On-demand history recovery](xref:OnDemandHistoryRecoveryConfiguration).<br>**CurrentOnly** - The adapter component operates normally and on-demand history recovery is disabled.<br>**CurrentWithBackfill** <sup>2</sup> - The adapter component operates normally, but disconnections are recorded based on device status. History recovery backfills data once device status is `good`. On-demand history recovery is disabled. For more information, see [Automatic history recovery](xref:AutomaticHistoryRecovery). <br>Changing the mode requires a restart of the adapter component.
+| **serverFailoverEnabled** | Optional | `boolean` | Specifies whether to use server-side failover. By default this is set to true and uses the server failover configuration. When set to false, the server failover configuration is ignored. <br><br> Allowed value: `true` or `false` <br> Default value: `true` |
```
Comment (derrick-at-aveva) [inline content/configuration/data-source.md:67]: >Specifies whether to use server-side failover. By default this is set to true and uses the server failover configuration.

Those 2 sentences can be deleted without any loss of information.

**Review by derrick-at-aveva — CHANGES_REQUESTED** [review]

File: content/configuration/data-source.md (hunk 1 above)
Comment (philliprlittle) [reply content/configuration/data-source.md]: I agree

**Review by philliprlittle — APPROVED** [review]
looks good

**Review by derrick-at-aveva — APPROVED** [review]

**Review by rmuolic — APPROVED** [review]

**Comment by bbilbrey-aveva** [comment]
This feature has been removed.

**Pull request closed without merging**